lib/packages/crypto/src/getBurstAddressFromAccountId.ts

```diff
@@ -1,8 +1,8 @@
-import { BurstUtil } from "./burstUtil";
+import { encode } from "@burstjs/util";
 
 /*
 * Convert the account id to the appropriate Burst address
 */
 export const getBurstAddressFromAccountId = (id: string): string => {
-    return BurstUtil.encode(id);
+    return encode(id);
```

Comment:
IMHO a nonsense function... is in util already...

Reply:
Nice catch!